Casks/l/longplay.rb

```diff
@@ -0,0 +1,26 @@
+cask "longplay" do
+  version "0.5.3,458"
+  sha256 "9257519386ff255b5a0146c7f74714558e642c7d8b8bc41189797afee4999f9e"
+
+  url "https://download.longplay.app/mac/longplay-#{version.csv.first}-#{version.csv.second}.zip",
+      verified: "download.longplay.app/"
+  name "Longplay"
+  desc "Rediscover, enjoy and organise your album collection"
```

Comment:
```suggestion
  desc "Album-focused music player"
```

We want to be as informative as possible here: [docs for reference](https://docs.brew.sh/Cask-Cookbook#stanza-desc).